test/python/importer/jit_ir/node_import/prim.py

```diff
@@ -14,12 +15,16 @@
 mb = ModuleBuilder()
 
 
+# Import TorchScript IR string as ScriptFunction
+def import_ts_ir(ts_ir_str):
+    cu = CompilationUnit()
+    mb.import_function(cu.create_function('main', torch._C.parse_ir(ts_ir_str)))
```

Comment:
nit: can we pass the function name as argument? And give a more descriptive name according to the specific test. In this case, it can be `__torch__.prim_Constant_list`.